tests/serialization/test_dag_serialization.py

```diff
@@ -1295,6 +1295,67 @@ class DummyTask(BaseOperator):
             '<TIDep(Trigger Rule)>',
         ]
 
+    def test_derived_dag_deps_sensor(self):
+        """
+        Tests DAG dependency detection for sensors, including derived classes
+        """
+        from airflow.operators.dummy import DummyOperator
```

Comment:
You will need to replace this import with `EmptyOperator`

Reply:
Thanks, I could not interpret the error message in the CI tests correctly